network/discovery_test.go

```diff
@@ -112,13 +112,13 @@ func TestRoutingTable_Disconnected(t *testing.T) {
 		1: defaultConfig,
 	}
 
-	servers, createErr := createServers(3, paramsMap)
+	servers, createErr := createServers(2, paramsMap)
```

Comment:
We simply didn't need more than 2 servers for be able to do this test or is it something else?

Reply:
Yes, we needed just 2 servers to test this functionality, the `3` was leftover from some other refactor